test/dlc_tests/conftest.py

```diff
@@ -864,6 +864,28 @@ def skip_inductor_test(request):
             )
 
 
+@pytest.fixture(autouse=True)
+def skip_torchdata_test(request):
+    if "training" in request.fixturenames:
+        img_uri = request.getfixturevalue("training")
+    elif "pytorch_training" in request.fixturenames:
+        img_uri = request.getfixturevalue("pytorch_training")
+    elif "inference" in request.fixturenames:
+        img_uri = request.getfixturevalue("inference")
+    elif "pytorch_inference" in request.fixturenames:
+        img_uri = request.getfixturevalue("pytorch_inference")
+    else:
+        return
+    _, image_framework_version = get_framework_and_version_from_tag(img_uri)
+    if request.node.get_closest_marker("skip_torchdata_test"):
+        if Version(image_framework_version) in SpecifierSet("<2.2"):
+            pytest.skip(
+                f"Torchdata has paused development as of July 2023 and the latest compatible PyTorch version is 2.1.1."
```

Comment:
Does it still support `2.2`? I see that we are skipping the tests for all versions `<2.2`.

Reply:
whoops. wrong logic.

Reply:
fixed it to skipping images greater than 2.1.1.